backend/pick-git/src/main/java/com/woowacourse/pickgit/post/application/PostService.java

```diff
@@ -48,7 +48,8 @@ public PostResponseDto write(PostRequestDto postRequestDto) {
             .findByBasicProfile_Name(postRequestDto.getUsername())
             .orElseThrow(() -> new IllegalArgumentException("해당하는 사용자가 없습니다."));
 
-        Post post = postRepository.save(new Post(postContent, getImages(postRequestDto), user));
+        Post post = postRepository.save(new Post(postContent, getImages(postRequestDto),
+            postRequestDto.getGithubRepoUrl(), user));
```

Comment:
```suggestion
        Post post = postRepository.save(
            new Post(postContent, getImages(postRequestDto),
            postRequestDto.getGithubRepoUrl(), user)
        );
```

이렇게 개행하면 보기 더 편할 것 같아요!